L1Trigger/L1THGCal/interface/HGCalAlgoWrapperBase.h

```diff
@@ -22,9 +22,15 @@ typedef HGCalAlgoWrapperBaseT<std::vector<edm::Ptr<l1t::HGCalTowerMap>>,
                               std::pair<const edm::EventSetup&, const edm::ParameterSet&>>
     HGCalTowerMapsWrapperBase;
 
+typedef HGCalAlgoWrapperBaseT<l1t::HGCalMulticlusterBxCollection,
+                              l1t::HGCalMulticlusterBxCollection,
+                              std::pair<const edm::EventSetup&, const edm::ParameterSet&>>
+    HGCalSortingTruncationWrapperBase;
```

Comment:
I would rather see the `WrapperBase` class to be named something like `HGCalStage2FilteringWrapperBase` ("filtering" being more generic than "SortingTruncation"), and then we can have different specialized class that can do a `SortingTruncation` or other kinds of filtering.

Reply:
Ok sure, changed.